7_drivers_munge/src/GCM_targets_wrapper.R

```diff
@@ -0,0 +1,10 @@
+build_GCM_pipeline <- function(ind_file, ...) {
+  # Build the targets pipeline
+  tar_make()
```

Comment:
Do you want this to make just one target instead of all? Seems if you include other things in _targets.R that aren't related to this part of the pipeline, you may not want those all to be built or checked.

Reply:
Yes, that's a good point. I was wondering about that on Wednesday. In this simple example, it works fine, but I agree it would be better to tie this more closely to the final products of our targets pipeline that do relate to this piece of the `lake-temp-model-prep` pipeline (e.g., the eventual processed driver files, the names of which we'll return in the object `pipelines_files_out`). I'll make that change now so that we're set up better for expanding this test pipeline.

Reply:
Hmmm that's interesting. So setting ourselves up to be able to have multiple `scipiper` targets that call `tar_make()`? That would add some nice flexibility. 

I see @hcorson-dosch made this change, but we probably want to rename now that the target out will be specifically tied to our GCM munging pipeline. I suggest this change in a future PR: `pipeline_files_out` >> `gcm_files_out`